docs/build.md

```diff
@@ -28,4 +30,27 @@ cd atomic-swap
 make build
 ```
 
-This creates `swapd`, `swapcli` and `relayer` binaries in the `bin` folder.
+This creates `swapd` and `swapcli` binaries in the `bin` folder.
+
+Note: if you wish to run a bootnode (see [here](./bootnode.md)), run `make build-all`.
+
+## Docker
+
+1. Ensure docker is installed on your machine.
+
+2. Build the docker image:
+```bash
+./scripts/docker/build-docker-image.sh 
```

Comment:
In my bootnode PR, I added a `make docker-images` option for this.  There are now 2 scripts (which that make target execute):
```
./scripts/docker-swapd/build-docker-image.sh
./scripts/docker-bootnode/build-docker-image.sh
```